master/getting-started/kubernetes/installation/vagrant/master-config.yaml

```diff
@@ -23,10 +38,6 @@ coreos:
         After=etcd-member.service
         [Service]
         TimeoutStartSec=1800
```

Comment:
Now that these services don't download the images, it doesn't feel right to me having these services installed if the image is not present.  Doesn't this mean the services will attempt to start when they can't and so we'll get various errors logged?  Might be confusing if that's the case.

If we have a download everything script, wouldn't it be better for that do be done as part of the installation rather than have it as a service?